File: app/src/main/java/ua/regin/wallpapers/entity/PreviewImage.java
package ua.regin.wallpapers.entity;

import java.io.Serializable;

public class PreviewImage implements Serializable {

    private String url;
    private int height;
    private int width;

    public String getUrl() {
        return url;
    }

    public int getHeight() {
        return height;
    }

    public int getWidth() {
        return width;
    }
}

File: app/src/main/java/ua/regin/wallpapers/entity/PreviewImage.kt
package ua.regin.wallpapers.entity;

import java.io.Serializable;

class PreviewImage : Serializable {

    var url: String;
    var height: Int;
    var width: Int;

    constructor(url: String, height: Int, width: Int) {
        this.url = url
        this.height = height
        this.width = width
    }
}
